posts/inside-rust/2024-12-17-project-director-update.md

```diff
@@ -0,0 +1,67 @@
+Rust Foundation][carol-nichols-board-announcement].
+
+[carol-nichols-board-announcement]: https://foundation.rust-lang.org/news/announcing-the-rust-foundation-s-newest-project-director-carol-nichols/
+
+One of my personal goals for my term as a Project Director is to communicate more and in different
+ways with the Rust Project and the wider Rust community about what it’s like being a Project
+Director and what the Rust Foundation is working on. This update post is one experiment along those
+lines.
+
+In this post, you’ll find the highlights, from my perspective, of last month’s board meeting now
+that the minutes have been posted publicly. I might cover other topics in the future, and I hope to
+encourage other Project Directors to write as well, but recapping Foundation Board meetings is
+where I’ve decided to start.
+
+The full board meeting minutes are [posted on the Foundation’s website][foundation-resources] after
+the following month’s board meeting where the minutes are approved. The [meeting on November 12,
+2024][foundation-board-minutes-2024-11], was the first board meeting I attended, and here’s what I
+thought were the most interesting parts.
+
+[foundation-resources]: https://foundation.rust-lang.org/resources/
+[foundation-board-minutes-2024-11]: https://foundation.rust-lang.org/static/minutes/2024-11-12-minutes.pdf
+
+The Foundation staff gave the board updates on their recent work:
+
+- Bec Rumbul, in the Executive Director update, covered that the reworked Trademark Policy draft
+  was in public comment. The Foundation is pursuing funding from the Sovereign Tech Fund and
+  [Alpha-Omega]. The Foundation is monitoring the situation with a few Linux maintainers’
+  permissions being removed due to US sanctions: there’s no similar situation involving Rust at
+  this time but Bec is preparing for the possibility.
+- Joel Marcey gave the Infrastructure/Technology update: Foundation employees are working on
+  reducing Rust’s CI costs, and there was progress on a number of other technical initiatives.
+- Paul Lenz’s Finance and Grants update included a draft 2025 budget, and a proposal for spending
+  $10,000 on a trial of an internship program which the board voted to approve.
+- Gracie Gregory gave a Communications update that included the securing of a venue for RustConf
+  2025 that will be publicly announced soon, and that the Foundation’s new website was almost ready
+  to launch.
+
+[Alpha-Omega]: https://alpha-omega.dev/
+
+Ryan Levick gave the Project Director update, which included discussion of the Project Goals
+Initiative working to define the Project’s priorities for the near future.
+
+The main item of business in the meeting was discussion of [DARPA’s Translating All C To Rust (aka
+TRACTOR) program][darpa-tractor] and the Rust Foundation’s possible involvement in the evaluation
+portion of the contest. Us Project Directors have some questions and thoughts surrounding this
```

Comment:
Not sure if you want to write this but it was a little more than questions and thoughts, there were actual concerns. That doesn't mean that they can't be addressed but were concerns.

Reply:
I'm going to leave this as-is. Thanks!